README.md

```diff
@@ -238,11 +238,34 @@ Simplified Login comes with the following localization support:
 1. 🇩🇰 Danish
 1. 🇬🇧 English (Default)
 
-#### Releasing
+## Local development
 
-1. Make sure all changes going in the release have been merged to `master` branch.
+### Setup
+
+## Prerequisites
```

Comment:
Wrong level of header? Should be `####`.
(Same for "Steps" and "Unit tests" below.)